src/main/scala/code/snippet/UserSnips.scala

```diff
@@ -292,12 +292,26 @@ object UserRegister extends ReCaptcha with SnippetHelper {
               User.sendEmailConfirmation(newUser)
               RedirectTo("/", () => S.notice("Un correo electronico ha sido enviado a su cuenta con instrucciones para acceder."))
             } else {
-              S.error("register_err", "Las contraseñas no coinciden")
+              S.error("password_err", "Las contraseñas no coinciden")
               Noop
             }
           case error: List[FieldError] =>
-            S.error(error)
-            Noop
+            error.foreach ( el => {
+              el match {
+                case email_id => 
+                  S.error("email_err", email_id.msg)
```

Comment:
Nope. id does not exist in el. at least, the compiler gives me that error message. 
Anyway, I am checking one by one because I want to write message below the corresponding field/s. Or do you prefer to show error messages into only one container?
After reviewing the fields name and lastName, It is not mandatory to use a name or lastName for registering users.. they can be empty. Is that ok?